probe-rs/src/bin/probe-rs/util/rtt.rs

```diff
@@ -464,9 +465,20 @@ impl RttActiveTarget {
                 // 3. Default with timestamp without location
                 // 4. Default without timestamp with location
                 // 5. Default without timestamp without location
+                let mut fileinfo = String::new();
                 let format = log_format.unwrap_or(match (show_location, has_timestamp) {
-                    (true, true) => "{t} {L} {s}\n└─ {m} @ {F}:{l}",
-                    (true, false) => "{L} {s}\n└─ {m} @ {F}:{l}",
+                    (true, true) => {
+                        fileinfo.push_str("{t} {L} {s}\n");
+                        fileinfo.push_str(&"└─ {m} @ {F}:{l}".dimmed());
```

Comment:
IIRC this won't work because the color information somehow gets lost when you go from `ColoredString` to `String`. I don't remember the details.

What you're trying to do should be supported by the defmt log formatter with this format string:
```suggestion
                        fileinfo.push_str(&"{└─ {m} @ {F}:{l}%dimmed}");
```
But the current version of defmt does not support this yet, we need a new release.